view/grpc/handlers/handler.go

```diff
@@ -41,6 +42,10 @@ func errorStatusGrpc(err error) error {
 		return status.Error(codes.NotFound, codes.NotFound.String())
 	}
 
+	if errors.Is(err, bcrypt.ErrMismatchedHashAndPassword) {
```

Comment:
this should be service error, 
today I use bcrypt, tomorrow something new, why do I need to go to change view layer if I want to change hashing algorithm